### PR TITLE
add info about prove_with_opts and different receipt types

### PR DESCRIPTION
**Pull request by sashaaldrick**
added a note to the `proving options` page to specify that users can use `prove_with_opts` to choose between a `composite`, `succinct` or `groth16` receipt.

### CODE REVIEW

File: website/api_versioned_docs/version-1.0/generating-proofs/proving-options.md [hunk 1]
```diff
@@ -18,6 +18,8 @@ Users can choose between:
 2. [Local Proving]: Proofs are generated on the user's own CPU/GPU.
 3. [Remote Proving]: Proofs are generated by [Bonsai], a highly parallelized, highly performant proving service.
 
+Furthermore, [prove_with_opts](https://docs.rs/risc0-zkvm/latest/risc0_zkvm/trait.Prover.html#method.prove_with_opts) allows users to choose between [composite, succinct or groth16 receipts](https://docs.rs/risc0-zkvm/latest/risc0_zkvm/enum.ReceiptKind.html).
```
Comment (pdg744) [inline website/api_versioned_docs/version-1.0/generating-proofs/proving-options.md:21]: also, please links to the bottom of the doc to make it easier to manage link maintenance.

File: website/api_versioned_docs/version-1.0/generating-proofs/proving-options.md [hunk 2]
```diff
@@ -18,6 +18,8 @@ Users can choose between:
...
+
 We recommend option 1 during development, option 2 for applications involving private inputs, and option 3 for all other applications.
```
[the remark this answers is not in the document]
Comment (sashaaldrick) [reply website/api_versioned_docs/version-1.0/generating-proofs/proving-options.md:23]: That's a great point. I've moved it below, and into a tip callout, just to segment it a bit better. Let me know if this helps.

File: website/api_versioned_docs/version-1.0/generating-proofs/proving-options.md (hunk 1 above)
Comment (sashaaldrick) [reply website/api_versioned_docs/version-1.0/generating-proofs/proving-options.md:21]: ty! ✅

[the remark this answers is not in the document]
Comment (sashaaldrick) [reply website/api_versioned_docs/version-1.0/generating-proofs/proving-options.md:21]: very good catch thanks, I'll remember this going ahead

[the remark this answers is not in the document]
Comment (pdg744) [reply website/api_versioned_docs/version-1.0/generating-proofs/proving-options.md]: let's get these changes duplicated into the`/api/` directory on this PR

**Comment by pdg744** [comment]
Looks good! Let's duplicate the change to the `/api/` directory before we land it.

**Comment by sashaaldrick** [comment]
I duplicated the changes to `next` as you mentioned @pdg744, only thing is that this also replicated the callout I added in a previous PR about the apple silicon and stark2snark issue.

What's your intuition about changes being version specific vs replicated in next? Just want to get some context to apply this going forward 🙏

**Comment by flaub** [comment]
I would think of it like all new content goes into `next` especially if it only applies to what's on `main`. If we backport it or we add new content that works for 1.0, then we'd want to add this to the 1.0 versioned docs.

**Comment by pdg744** [comment]
Agreed with what Frank said. I shouldn't have approved your PR with the stark2snark issue until you had included the changes on `next` -- my bad :)

**Review by pdg744 — APPROVED** [review]
Thanks Sasha!

**Comment by sashaaldrick** [comment]
@flaub @pdg744 thanks for the context, that makes sense

**Pull request merged**